es-app/src/CollectionSystemManager.cpp

```diff
@@ -208,6 +208,7 @@ void CollectionSystemManager::updateSystemsList()
 	{
 		if ((*sysIt)->isCollection())
 		{
+			(*sysIt)->getIndex()->setUIModeFilters();
```

Comment:
@zigurana any objections or considerations to its use here - or anywhere in particular? I take it that this just enforces the UI-mode-specific filters, correct?

Best.

Reply:
Yes, that's correct, but now that I look at it, why should you (as in CollectionSystemManager) care? maybe we should add it to ViewController->getGamelistview(), let me think about that for a bit.